addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -645,6 +675,41 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
+    private boolean handleEqualizerCommands(String channelId, Command command, Connection connection, Device device)
+            throws URISyntaxException {
+        if (command instanceof RefreshType) {
+            this.lastKnownEqualizer = null;
+        }
+        if (command instanceof DecimalType) {
+            DecimalType value = (DecimalType) command;
+            if (this.lastKnownEqualizer == null) {
+                updateEqualizerState();
+            }
+            JsonEqualizer lastKnownEqualizer = this.lastKnownEqualizer;
+            if (lastKnownEqualizer != null) {
+                JsonEqualizer newEqualizerSetting = lastKnownEqualizer.createClone();
+                if (channelId.equals(CHANNEL_EQUALIZER_BASS)) {
```

Comment:
Not really relevant in this case but imaybe for later to improve the code. In general you should position string literals first to avoid a `null` check.